### ISSUE TITLE
Roll Table armor and weapon qualities missing.

### ISSUE THREAD

**Issue by Tireddadgamer** [baileymh/pf1e-content#52]
I've done my best to see if this is the right place to mention this or if I need to reach out to someone else, but I'm honestly not sure. Please forgive me if I'm wrong.

Loving the access to the PF1 content on foundry first of all, thanks for your work.

Have a question about the roll tables for magical items. There seems to be a lot missing from the weapon/armor/etc qualities when it rolls, for example. Every time a magical armor is rolled, say from the "greater minor magic item" list, it only ever rolls "warding". The list of +1 special qualities only has warding. The same is true.

it is quite possible I failed to load something correctly. Its also possible that it just hasnt been added yet, if so, I will happily button my lip and wait. Either way, I appreciate whatever you can do to help me figure this out.

**Comment by Klala** [comment]
Yes. I just noticed this myself. Seems like the tables are incomplete.
